lib/matplotlib/tests/test_axes.py

```diff
@@ -1891,6 +1891,9 @@ def test_stairs_update(fig_test, fig_ref):
     assert np.allclose(h.get_data()[1], np.arange(4))
     h.set_baseline(-2)
     assert h.get_baseline() == -2
+    h.set_baseline([-2, -2, -2])
+    assert np.array_equal(h.get_baseline(), np.array([-2, -2, -2]))
```

Comment:
The test is currently not testing anything, see #18579. We probably should try to get that fixed first.